server/graphql/v2/mutation/TierMutations.js

```diff
@@ -130,10 +132,12 @@ const tierMutations = {
         tierCreateData.data.invoiceTemplate = args.tier.invoiceTemplate;
       }
 
+      const tier = await models.Tier.create(tierCreateData);
+
```

Comment:
@Betree : The reason I did this change was that I noticed that otherwise when the tier creation or tier edit modal is saved you need to do a page refresh sometimes to notice the changes. This is because previously we were doing the update after the cache is cleared.